src/bin/init.rs

```diff
@@ -47,12 +54,37 @@ pub fn execute(args: &ArgMatches) -> Result<()> {
         builder.create_gitignore(true);
     }
 
+    if let Some(author) = get_author_name() {
+        debug!("Obtained user name from gitconfig: {:?}", author);
+        config.book.authors.push(author);
```

Comment:
While we're adding the author's name, should we think about giving the book a default title (or even prompting for one) for the user?

Reply:
Can definitely add this one in there! 😄